### ISSUE TITLE
bug(mat-select): Accessibility scan showing Orphaned Form Label failure when correctly using the component

### ISSUE THREAD

**Issue by csirota97-a11y** [angular/components#32735]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

I am using 18.2.14; however, this issue also present on the https://material.angular.dev/components/select/overview page.

When using the WAVE accessibility scanner, we see that every mat-select present on the page flags an "Orphaned Form Label" accessibility failure.

### Reproduction

REPL: https://material.angular.dev/components/select/examples
Steps to reproduce:
1. Install the WAVE browser extenstion https://wave.webaim.org/extension/
2. Navigate to: https://material.angular.dev/components/select/examples
3. Right click on page and select "WAVE this page"
4. Notice the "Orphaned Form Label" alerts on all mat-select

*Note - I tried to upload a screenshot, but the file upload seems to be failing for some reason

<img width="1735" height="1105" alt="Image" src="https://github.com/user-attachments/assets/b60ea6b5-6b3e-4072-a947-1f6e22f88d24" />

### Expected Behavior

Labels should be correctly programatically associated to the form elements so that the WAVE scan does not flag any correctly used mat-select components as "Orphaned Form Label"

### Actual Behavior

Labels are incorrectly associated to the form elements so that the WAVE scan is flagging correctly used mat-select components as "Orphaned Form Label"

### Environment

- Angular: 18.2.12
- CDK/Material: 18.2.14-21.1.2
- Browser(s): Edge, Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS Sequoia, Windows 11

**Comment by SHUBHAM-ACHARYA** [comment]
Hi @csirota97-a11y ,
Did you get any fix for this orphaned form label?

I'm having the same problem in my project

**Comment by csirota97-a11y** [comment]
Unfortunately, there's been no resolution

**Comment by crisbeto** [comment]
This is intentional since other tools complain if the `label` is associated with a non-native control like `mat-select`. We may be able to do something with a `role` or by rendering it as a `div`, but it's also worth noting that we run accessibility tests occasionally and this has never caused issues for an actual user.

**Comment by csirota97-a11y** [comment]
Thank you for the detailed explanation.From: Kristiyan Kostadinov ***@***.***>Date: Tuesday, February 17, 2026 at 1:57 AMTo: angular/components ***@***.***>Cc: csirota97-a11y ***@***.***>, Mention ***@***.***>Subject: Re: [angular/components] bug(mat-select): Accessibility scan showing Orphaned Form Label failure when correctly using the component (Issue #32735)Get 							Outlook for Mac					crisbeto left a comment (angular/components#32735)This is intentional since other tools complain if the label is associated with a non-native control like mat-select. We may be able to do something with a role or by rendering it as a div, but it's also worth noting that we run accessibility tests occasionally and this has never caused issues for an actual user.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
